util/ts_set.go

```diff
@@ -60,7 +61,7 @@ func (s *TSSet) Put(tss ...uint64) {
 func (s *TSSet) GetAll() []uint64 {
 	s.RLock()
 	defer s.RUnlock()
-	if len(s.m) == 0 {
+	if s.m == nil || len(s.m) == 0 {
```

Comment:
This change looks unnecessary because `len(nil map)`  also returns 0

Reply:
Done